app/src/main/res/values-fr/strings.xml

```diff
@@ -69,17 +72,19 @@
     <string name="logs_cannot_read">Lecture du journal impossible: \n</string>
     <string name="menuReload">Recharger</string>
     <string name="logs_clear_failed_2">Échec d\’effacement du journal</string>
-    <string name="not_logcat_2">S\’il vous ai demandé un Logcat Android, alors vous êtes dans la mauvaise section. Ici il s\’agit du journal LSPosed; si vous avez besoin d\’un logcat, vous pourrez trouver ce dont vous avez besoin sur XDA.</string>
+    <string name="verbose_log_not_avaliable">Le journal verbeux n’est pas disponible, si vous l\’avez activé sans avoir redémarré, essayez à nouveau après avoir redémarrer.</string>
 
     <!-- Notification -->
     <string name="module_is_not_activated_yet">Le module LSPosed n\’est pas encore actif</string>
     <string name="module_is_not_activated_yet_detailed">%s à été installé, mais il n\’est pas encore installé</string>
-    <string name="xposed_module_updated_notification_content">%s à été mis à jour</string>
+    <string name="xposed_module_updated_notification_title">Module Xposed mis à jour</string>
+    <string name="xposed_module_updated_notification_content">%s à été mis à jour, merci de forcer l\’arrêt ou de redémarrer les applis dans leurs champs d\’application</string>
+    <string name="xposed_module_updated_notification_title_system">Module Xposed à jour, redémarrage du système requis</string>
+    <string name="xposed_module_updated_notification_content_system">%s à été mis à jour, puisque le champ d'application contient le Sous Système, le redémarrage est nécessaire pour appliquer les changements</string>
```

Comment:
\\'

Reply:
I think i just fix it now in my last update on my repo